rt/src/hand_off.rs

```diff
@@ -0,0 +1,51 @@
+// Licensed under the Apache-2.0 license.
+
+const FHT_MAGIC: u32 = 0x54484643;
+#[repr(C)]
+#[derive(Clone, Debug, Default)]
+pub struct HandoffParams {
+    magic: u32,
+    major: u16,
+    minor: u16,
+    mft_base_addr: u32,
+    fips_base_addr: u32,
+    rt_base_addr: u32,
+    fmc_cdi_kv_idx: u8,
+    fmc_priv_key_idx: u8,
+    fmc_pub_key_idx: u8,
+    fmc_cert_dv_idx: u8,
+    rt_cdi_kv_idx: u8,
+    rt_priv_key_kv_idx: u8,
+    rt_pub_key_dv_idx: u8,
+    rt_cert_dv_idx: u8,
+    reserved: [u8; 20],
+}
+
+impl HandoffParams {
+    pub fn is_valid(&self) -> bool {
+        self.magic == FHT_MAGIC
+    }
+}
+
+impl Default for HandoffParams {
+    // TODO : replace placholder with actual values.
```

Comment:
values will be populated by ROM/FMC as they set things up. There should not be a set of defaults already built in. The Indexes should start as 0xFF to indicate they are not valid (use something like INVALID_IDX = 0xFF) and all others should be set to 0. Nothing else should be assumed by FMC or Runtime FW

Reply:
This data structure now resides in the common crate in main branch.